### PR TITLE
Harmonic mag dipole

### PR DESCRIPTION
**Pull request by thast**
- .rst pages
- some fix

### CODE REVIEW

File: content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/frequency_domain/dipole_sources/electric_dipole/analytic_solution.rst [hunk 1]
```diff
@@ -8,3 +8,81 @@ Analytic Solution
     Purpose here
```
Comment (lheagy) [inline content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/frequency_domain/dipole_sources/electric_dipole/analytic_solution.rst]: could you add a purpose statement here? Also, there are lots of equations, any that should be in the equation bank?

File: content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/frequency_domain/dipole_sources/electric_dipole/asymptotics.rst [hunk 2]
```diff
@@ -8,9 +8,99 @@ Asymptotics
     Purpose here
```
Comment (lheagy) [inline content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/frequency_domain/dipole_sources/electric_dipole/asymptotics.rst]: could you add a purpose statement here please?

**Comment by sgkang** [comment]
Can we merge this? I want to put dipole stuff into a branch.
Now we have four different branch. Any thoughts? @lheagy @thast @micmitch ?

**Comment by lheagy** [comment]
before merging, could you 
- [ ] add the `underconstruction.html` banner to the top of pages that are not finished

```
.. raw:: html 
    :file: ../../underconstruction.html
```
(underconstruction.html is in the root em directory, so the path will need to be adjusted appropriately)

**Pull request merged**